tests/foreman/cli/test_logging.py

```diff
@@ -230,3 +230,29 @@ def test_positive_logging_from_dynflow(module_org):
                 assert match, "Request ID not found"
                 logger.info("Request ID found for logging from dynflow ")
     assert POST_line_found, "The POST command to enable a repo was not found in logs."
+
+
+@pytest.mark.tier4
+@pytest.mark.stubbed
+@pytest.mark.skip_if_open("BZ:1732522")
+def test_positive_logging_verify_pulp3_logs():
```

Comment:
```suggestion
def test_positive_logging_from_pulp3():
```
I think to many words with similar meanings